Sources/BugsnagPerformance/Public/BugsnagPerformanceConfiguration.mm

```diff
@@ -80,4 +87,9 @@ - (BOOL)isValidApiKey:(NSString *)apiKey {
     return isHex && [apiKey length] == BSGApiKeyLength;
 }
 
+- (BOOL)shouldSendReports {
+    return self.enabledReleaseStages.count == 0 ||
+           [self.enabledReleaseStages containsObject:self.releaseStage ?: @""];
+}
```

Comment:
I think it would be cleaner to make this decision inside `BugsnagPerformanceImpl`. That way, `BugsnagPerformanceConfiguration`'s responsibility is solely to record what the user wanted, and `BugsnagPerformanceImpl` is responsible for doing something about it.